Source/msg.cpp

```diff
@@ -3439,6 +3455,8 @@ size_t ParseCmd(size_t pnum, const TCmd *pCmd)
 		return OnFriendlyMode(pCmd, player);
 	case CMD_SYNCQUEST:
 		return OnSyncQuest(pCmd, pnum);
+	case CMD_CHEAT_REMOVE_EXPERIENCE:
+		return OnCheatRemoveExperience(pCmd, pnum);

```

Comment:
Could we do this without sending a network message? Even move CMD_CHEAT_EXPERIENCE ot of here as well?